app/src/main/java/com/kamron/pogoiv/Pokefly.java

```diff
@@ -279,10 +291,56 @@ public int onStartCommand(Intent intent, int flags, int startId) {
             createIVButton();
             createArcPointer();
             createArcAdjuster();
+            /* Assumes MainActivity initialized ScreenGrabber before starting this service. */
```

Comment:
Actually, no, this isn't one of those hacks. This is intentional. I couldn't move it to Pokefly because getting the MediaProjection mProjection object needs some intent handling and also showing the "Grant permissions" stuff in the UI.

Pokefly has no UI. So, I couldn't move it there. I'm not sure if mProjection can be passed through an intent. If it can, then we can remove this assumption.